src/psyclone/psyGen.py

```diff
@@ -1390,21 +1390,24 @@ def update(self):
 
 
 class Schedule(Node):
-    '''
-    Stores schedule information for a sequence of statements
+    ''' Stores schedule information for a sequence of statements.
 
-    :param sequence: the sequence of PSyIR nodes that the schedule execute.
+    :param sequence: the sequence of PSyIR nodes that make up the schedule.
     :type sequence: :py:class:`psyclone.psyGen.Node`
     :param parent: that parent of this node in the PSyIR tree.
-    :type parent: :py:class:`psyclone.psyGen.Node`
+    :type parent: list of :py:class:`psyclone.psyGen.Node`
     '''
 
     def __init__(self, sequence, parent):
         Node.__init__(self, children=sequence, parent=parent)
 
     @property
     def dag_name(self):
-        ''' Return the name to use in a dag for this node'''
+        ''' Return the name to use in a dag for this node.
```

Comment:
You can just get rid of the first line of this docstring now - it's all covered by the Sphinx mark-up below.

Reply:
done